File: app/src/main/java/studio/mandysa/music/logic/Model/PlaylistModel.java
package studio.mandysa.music.logic.Model;

import mandysax.anna2.annotation.Array;
import mandysax.anna2.annotation.Value;

@Array
public class PlaylistModel {
    @Value("id")
    public String id;

    @Value("name")
    public String name;

    @Value("copywriter")
    public String info;

    @Value("picUrl")
    public String picUrl;

}

File: app/src/main/java/studio/mandysa/music/logic/model/PlaylistModel.kt
package studio.mandysa.music.logic.model

import mandysax.anna2.annotation.Value

/**
 * @author liuxiaoliu66
 */
class PlaylistModel {

    @Value("result")
    val playlist: List<Playlist>? = null

    class Playlist {
        @Value("id")
        var id: String? = null

        @Value("name")
        var name: String? = null

        /*@Value("copywriter")
        public String info;*/
        @Value("picUrl")
        var picUrl: String? = null
    }

}